# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1204889050 18000
#      Fri Mar 07 06:24:10 2008 -0500
# Node ID 8bf1bcb0ad8fb31b97f0ce2a9d5f5d0a94e4e55b
# Parent  369b16e7f04b8b69b9193fbedce1d2516f1c7ba8
set axes layer property to "top"

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,10 +1,12 @@
 2008-03-07  John W. Eaton  <jwe@octave.org>
 
+	* plot/contourf.m: Set axes layer property to "top".
+
 	* plot/__go_draw_axes__.m: Handle tickdir property.
 
 2008-03-06  John W. Eaton  <jwe@octave.org>
 
 	* plot/hist.m: Avoid temporaries.
 	Allow matrix arguments when number of bins > 30.
 	From Robert S. Mahurin <rob@utk.edu>.
 
diff --git a/scripts/plot/contourf.m b/scripts/plot/contourf.m
--- a/scripts/plot/contourf.m
+++ b/scripts/plot/contourf.m
@@ -175,16 +175,18 @@ function varargout = contourf (varargin)
   endfor
 
   if (min (lev) == max (lev))
     set (gca (), "clim", [min(lev)-1, max(lev)+1]);
   else
     set (gca(), "clim", [min(lev), max(lev)]);
   endif
 
+  set (gca (), "layer", "top");
+
   if (nargout > 0)
     varargout{2} = h;
     varargout{1} = c;
   endif
 
 endfunction
 
 function [X, Y, Z, lvl, patch_props] = parse_args (arg)
